lib/src/Base/Stat/CovarianceModelImplementation.cxx

```diff
@@ -290,26 +296,31 @@ struct CovarianceModelDiscretizePolicy
       const UnsignedInteger iLocal = i - (jLocal * (jLocal + 1)) / 2;
       const UnsignedInteger iBase = iLocal * outputDimension_;
       const CovarianceMatrix localCovariance(model_(input_[iLocal], input_[jLocal]));
-      for (UnsignedInteger ii = 0; ii < outputDimension_; ++ii)
-        for (UnsignedInteger jj = 0; jj < outputDimension_; ++jj)
-          output_(iBase + ii, jBase + jj) = localCovariance(ii, jj);
+      for (UnsignedInteger jj = 0; jj < outputDimension_; ++jj)
+        for (UnsignedInteger ii = 0; ii < outputDimension_; ++ii)
+          output_(iBase + ii, iBase + jj) = localCovariance(ii, jj);
```

Comment:
@dbarbier is this line ok ? it seems jBase is no more used.

Reply:
Good catch, thanks.

Reply:
I am going to fix master